annotations.md

```diff
@@ -0,0 +1,25 @@
+# Annotations
+This OPTIONAL property MUST use the [annotation rules](annotations.md#rules).
+
+This property contains arbitrary metadata.
+
+## Rules
+
+Annotations MUST be a key-value map where both the key and value MUST be strings.
+While the value MUST be present, it MAY be an empty string.
+Keys MUST be unique within this map, and best practice is to namespace the keys.
+Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
+Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by other specifications and extensions.
+If there are no annotations then this property MUST either be absent or be an empty map.
+Implementations that are reading/processing manifest lists MUST NOT generate an error if they encounter an unknown annotation key.
+
+See [Pre-Defined Annotation Keys](manifest.md#pre-defined-annotation-keys).
+
+## Pre-Defined Annotation Keys
+
+This specification defines the following annotation keys, which MAY be used by manifest list and image manifest authors:
```

Comment:
Is there a reason to restrict the scope of the pre-defined keys to manifests and manifest lists?  For example, `org.opencontainers.source` seems like it would be a good match for a descriptor referencing layer tar archive.  If that's too big a shift for this PR, I'm fine spinning it off into a follow-up issue/PR.